platform-sdk/docs/proposals/metric-labels/README.md

```diff
@@ -0,0 +1,175 @@
+# Metric labels
+
++author: @hendrikEbbers
++status: draft
+
+## Summary
+
+This proposal describes a new feature for the metrics module.
+The feature is called "metric labels" and allows to add labels to metrics.
+Labels are key-value pairs that can be used to filter and group metrics.
+Why this proposal uses the name "metric labels" (like prometheus does) other libraries (like micrometer) use the term tags.
+
+## Motivation
+
+Today the metrics module only supports the definition of metrics without adding custom labels.
+While each metric has a  hardcoded category, name, and nodeId no other custom information can be added.
+This proposal suggests adding a new concept called "metric labels" to the metrics module and
+migration the `category`, `name`, and `nodeId` information to labels.
+While the category and name are defined as String values the nodeId is a class called `NodeId`.
+By doing so the metrics module becomes more flexible and can be used 100% independent of the
+platform module since it does not depend on the platform module's `nodeId` anymore.
+
+## Goals
+
+- Add a new concept called "metric labels" to the metrics module
+- Migrate the `category`, `name`, and `nodeId` information to labels
+- Make the metrics module independent of the platform module
```

Comment:
This is IMO the main benefit of adding labels as a generally available feature in the framework.